less/home-page.less

```diff
@@ -39,23 +84,53 @@
   }
 
   .content-destination {
-    width: 75%;
-    margin: 0 auto 30px;
+    padding: 3% 0 3% 10%;
+    border-bottom: 2px dashed @navigation-border;
+  
+    @media @mobile {
+      width: 100%;
+      margin: 5% 0 0 0;
+      padding: 0 5% 5% 0;
+    }
 
     img {
       border-radius: 10px;
     }
+
+    p {
+      width: 80%;
+
+      @media @mobile {
+        width: 110%;
```

Comment:
really cool usage of 110% width to get more text on the lines of the p tags in content-destination for the mobile view.